### PR TITLE
Feature/138523 charges and chargelinks test data

### PR DESCRIPTION
**Pull request by kristianschneider**
This provides the initial functionality for parsing Charges from a csv file

### CODE REVIEW

File: source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
```
Comment (mknic) [inline source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs:1]: License section is missing

**Review by mknic — APPROVED** [review]
LGTM. You just to add the license section to a file as mentoned :+1:

File: source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using System.Collections.Generic;
+using System.Globalization;
```
Comment (mknic) [inline source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs:3]: Unused references. Should be removed.

File: source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+using System.IO;
+using System.Text;
```
Comment (mknic) [inline source/test-data-generation/GreenEnergyHub.Aggregation.TestData.Application/Parsers/ChargeTestDataParser.cs:5]: Unused references. Should be removed.

**Review by johevemi — APPROVED** [review]

**Pull request merged**